### ISSUE TITLE
Create `deploy-release` public action

### ISSUE THREAD

**Issue by dequejenn** [dequelabs/axe-api-team-public#90]
### Overview ###

**Repo:** https://github.com/dequelabs/axe-api-team-public
**GHA Name:**  deploy-release
**Purpose:** To make sure everything finishes up automatically when a deployment has been completed
**Usage:** This GHA will be called from each repo when teh release pr is merged into master/main

**Inputs:**  
From repo:
- base = release
- head = main/master

**Outputs:**
- None

### Details ###

This action should 

1.  Get the version number using https://github.com/dequelabs/axe-api-team-public/pull/64
2. Get the commits using .github/actions/generate-commit-list-v1
3. Run through and label all issues closed in this release using .github/actions/label-and-move-released-issues-v1 (this should create a label with the version number if it doesn't already exist, label any issue closed in this version, and automatically move any issues into the `released` column if it was `dev done` or `done`)
4. Sync the master/main branch back into develop
Some of the repos already have this which can just be added into this so it is in a single place in case we need to make changes in the future
```
jobs:
  create_sync_pull_request:
    runs-on: ubuntu-latest
    steps:
      - uses: dequelabs/action-sync-branches@v1
        with:
          github-token: ${{ secrets.PAT }}
          pr-title: 'chore: merge master into develop'
          pr-body: 'Remember to _merge_ (rather than squash) this PR! Also, auto-merge does not work because many of the required checks do not run on the "master" branch.'
          pr-team-reviewers: axe-api-team
          head: master
```

The action should also have unit tests and a Readme. See https://github.com/JasonEtco/create-an-issue for template on how to do this.

Each of the public actions in https://github.com/dequelabs/axe-api-team-public should include a readme with information on how to use the action and a list all inputs and outputs.

**Comment by straker** [comment]
Blocked by https://github.com/dequelabs/axe-api-team-public/issues/91
